.github/workflows/release.yml

```
@@ -24,59 +28,33 @@ jobs:
         with:
           gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
           passphrase: ${{ secrets.GPG_PASSPHRASE }}
-
-      - name: Install spacectl
-        uses: spacelift-io/setup-spacectl@main
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: List versions for preprod
-        run: |
-          spacectl provider list-versions --type=spacelift
-        env:
-          SPACELIFT_API_KEY_ENDPOINT: https://spacelift-io.app.spacelift.dev
-          SPACELIFT_API_KEY_ID: ${{ secrets.SPACELIFT_PREPROD_API_KEY_ID }}
-          SPACELIFT_API_KEY_SECRET: ${{ secrets.SPACELIFT_PREPROD_API_KEY_SECRET }}
-
-      - name: List versions for prod
-        run: |
-          spacectl provider list-versions --type=spacelift
-        env:
-          SPACELIFT_API_KEY_ENDPOINT: https://spacelift-io.app.spacelift.io
-          SPACELIFT_API_KEY_ID: ${{ secrets.SPACELIFT_PROD_API_KEY_ID }}
-          SPACELIFT_API_KEY_SECRET: ${{ secrets.SPACELIFT_PROD_API_KEY_SECRET }}
-
-      - name: Set up release notes
-        run: |
-          echo ${{ steps.import_gpg.outputs.fingerprint }} > key.fingerprint
-          gpg --armor --export ${{ steps.import_gpg.outputs.keyid }} > key.asc
         
-      - name: Run GoReleaser
+      - name: Run GoReleaser to release to GitHub/HashiCorp registry
         uses: goreleaser/goreleaser-action@v2
```

might as well bump this to `v4` as well?
```suggestion
        uses: goreleaser/goreleaser-action@v4
```

---

In principle, good idea, but can we do it in a separate PR, just in case?

---

sure